docs/specifications/tests/Consistency-TP/consistency03.md

```diff
@@ -6,15 +6,8 @@
 
 ### Objective
 
-All authoritative name servers must serve the same SOA record for the
-designated domain. As per section 3.3.13 of [RFC 1035](https://tools.ietf.org/html/rfc1035),
-the field "REFRESH" in the SOA RDATA refers to the 32 bit time interval before the
-zone should be refereshed. The field "RETRY" refers to the 32 bit interval
-before a failed refresh should be retried, the field "EXPIRE" refers to a 32
-bit time value that specifies the upper limit on the time interval that can
-elapse before the zone is no longer authoritative and the field "MINIMUM" is
-the unsigned 32 bit minimum TTL field that should be exported with any
-resource record from the domain's zone
+All SOA record fields except SERIAL, RNAME and MNAME must be consistent
```

Comment:
I do not like the wording. It sounds like SERIAL, RNAME and MNAME does not have to consistent. I suggest
```
The SOA record fields REFRESH, RETRY, EXPIRE and MINIMUM must be consistent across all name servers authoritative for the tested zone.
```

Reply:
It is quite correct that Consistency03 disregards any and all inconsistency in SERIAL, RNAME and MNAME.

I'm working towards expanding this test case to include checks for RNAME and MNAME consistency. When those checks are included the RNAME and MNAME fields will be dropped from the list of exceptions. At that point we will have one test case for "only SERIAL" and one for "everything but SERIAL". That way it's clearer to the reader that these two test cases together cover the entire set of SOA RR fields.

Reply:
Added a note below about additional consistency tests for the remaining fields.